Wikipedia/Code/PlacesViewController.swift

```diff
@@ -114,7 +114,7 @@ class PlacesViewController: ArticleLocationCollectionViewController, UISearchBar
 
     override func viewDidLoad() {
 
-        listViewController = ArticleLocationCollectionViewController(articleURLs: [], dataStore: dataStore, contentGroup: nil, theme: theme)
+        listViewController = ArticleLocationCollectionViewController(articleURLs: [], dataStore: dataStore, contentGroup: nil, theme: theme, source: .places)
```

Comment:
@mazevedofs 

(Reason for this bug - https://github.com/wikimedia/wikipedia-ios/pull/5187/files#r1946990495)

You'll also want to set `self.source = .places` (if you can, may need to set the superclass property as mutable. That's how you can target the iPhone list view.

Reply:
Do you mean on ArticleVC? Or from ArticleLocationCollectionVC?